maracluster/1.02.1/Dockerfile

```diff
@@ -0,0 +1,32 @@
+################## BASE IMAGE ######################
+FROM ubuntu:18.04
+
+################## METADATA ######################
+LABEL base_image="ubuntu:18.04"
+LABEL version="1"
+LABEL software="maracluster"
+LABEL software.version="1.02.1"
+LABEL about.summary="MaRaCluster - a tool for clustering and merging of fragment spectra from shotgun proteomics experiments."
+LABEL about.home="https://github.com/statisticalbiotechnology/maracluster"
+LABEL about.documentation="https://github.com/statisticalbiotechnology/maracluster"
+LABEL about.license_file="https://github.com/statisticalbiotechnology/maracluster/blob/master/LICENSE"
+LABEL about.license="SPDX:Apache-2.0"
+LABEL about.tags="Proteomics"
+
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y g++ build-essential zlib1g-dev libbz2-dev libboost-all-dev wget
```

Comment:
Why g++ build etc... just need wget to get the deb files. Others are not needed and will increase container size 

Other expected libs should be anyway installed automatically alobg deb file

Reply:
if I leave the `wget` only in local tests it doen't work. I guess the deb is not installing the dependencies?

Reply:
exact error: 
```
shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
```

Reply:
Libraries removed and it is working. Only needs that remaining library.